lifechanger/src/app/footer/footer.component.html

```diff
@@ -0,0 +1,192 @@
+                    fill="#42446E"
+                  />
+                </svg>
+              </a>
+            </li>
+
+            <li>
+              <a href="https://www.linkedin.com/in/oksana-kolisnyk-0b9632247">
+                <svg
+                  class="nav-svg"
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="30"
+                  height="30"
+                  viewBox="0 0 30 30"
+                  fill="none"
+                >
+                  <g clip-path="url(#clip0_10_208)">
+                    <path
+                      d="M15 0.600098C7.04701 0.600098 0.600006 7.0471 0.600006 15.0001C0.600006 22.9531 7.04701 29.4001 15 29.4001C22.953 29.4001 29.4 22.9531 29.4 15.0001C29.4 7.0471 22.953 0.600098 15 0.600098ZM11.475 20.9686H8.55901V11.5846H11.475V20.9686ZM9.99901 10.4326C9.07801 10.4326 8.48251 9.7801 8.48251 8.9731C8.48251 8.1496 9.09601 7.5166 10.0365 7.5166C10.977 7.5166 11.553 8.1496 11.571 8.9731C11.571 9.7801 10.977 10.4326 9.99901 10.4326ZM22.125 20.9686H19.209V15.7681C19.209 14.5576 18.786 13.7356 17.7315 13.7356C16.926 13.7356 16.4475 14.2921 16.236 14.8276C16.158 15.0181 16.1385 15.2881 16.1385 15.5566V20.9671H13.221V14.5771C13.221 13.4056 13.1835 12.4261 13.1445 11.5831H15.678L15.8115 12.8866H15.87C16.254 12.2746 17.1945 11.3716 18.768 11.3716C20.6865 11.3716 22.125 12.6571 22.125 15.4201V20.9686Z"
+                      fill="#42446E"
+                    />
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_10_208">
+                      <rect width="30" height="30" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="1209"
+    height="5"
+    viewBox="0 0 1211 3"
+    fill="none"
+  >
+    <path
+      opacity="0.3"
+      d="M1 1.84112H1210"
+      stroke="#666666"
+      stroke-width="2"
+      stroke-linecap="square"
+    />
+  </svg>
+  <div class="footer2-container">
+    <div class="footer-primary">
+      <div>
+        <ul class="footer-links">
+          <li>
+            <a href="#home" class="footer-link">Home</a>
+          </li>
+          <li>
+            <a href="#about" class="footer-link">About</a>
+          </li>
+          <li>
+            <a href="#stack" class="footer-link">Technologies</a>
+          </li>
+          <li>
+            <a href="#projects" class="footer-link projects">Projects</a>
+          </li>
+
+          <li>
+            <a href="#contact" class="footer-link contact">Contact</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <p class="footer-p">
```

Comment:
i don't think you should have here class footer-p.  I see only one p tag in this html file. Probably will be better to set css selector as 
.footer {
p{
//some-styles
}
}

Reply:
Dear Mentor, thank you. I deleted the .footer-p class and instead used .footer p, .footer p span